.gitignore

```diff
@@ -138,3 +138,6 @@ tests/data/output
 
 # PyCharm
 .idea/
+
+# Put your own experiments here
+brew/*
```

Comment:
Please remove this addition to minimize the diff, it does not seem to add value and it will start our checks/ci again which will hopefully be green.

Reply:
done